test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationTests.cs

```diff
@@ -75,34 +76,76 @@ private OpenIdConnectConfiguration RunOpenIdConnectConfigurationTest(object obj,
+            Assert.NotNull(configuration.ClaimTypesSupported);
+            Assert.NotNull(configuration.DisplayValuesSupported);
+            Assert.NotNull(configuration.GrantTypesSupported);
+            Assert.NotNull(configuration.IdTokenEncryptionAlgValuesSupported);
+            Assert.NotNull(configuration.IdTokenEncryptionEncValuesSupported);
+            Assert.NotNull(configuration.IdTokenSigningAlgValuesSupported);
+            Assert.NotNull(configuration.RequestObjectEncryptionAlgValuesSupported);
+            Assert.NotNull(configuration.RequestObjectEncryptionEncValuesSupported);
+            Assert.NotNull(configuration.RequestObjectSigningAlgValuesSupported);
+            Assert.NotNull(configuration.ResponseModesSupported);
+            Assert.NotNull(configuration.ResponseTypesSupported);
+            Assert.NotNull(configuration.ScopesSupported);
             Assert.NotNull(configuration.SigningKeys);
+            Assert.NotNull(configuration.SubjectTypesSupported);
+            Assert.NotNull(configuration.TokenEndpointAuthMethodsSupported);
+            Assert.NotNull(configuration.TokenEndpointAuthSigningAlgValuesSupported);
+            Assert.NotNull(configuration.UILocalesSupported);
+            Assert.NotNull(configuration.UserInfoEndpointEncryptionAlgValuesSupported);
+            Assert.NotNull(configuration.UserInfoEndpointEncryptionEncValuesSupported);
+            Assert.NotNull(configuration.UserInfoEndpointSigningAlgValuesSupported);
         }
 
-        [Fact(DisplayName = "OpenIdConnectMetadataTests: GetSets")]
+        [Fact(DisplayName = "OpenIdConnectConfigurationTests: GetSets")]
         public void GetSets()
         {
             OpenIdConnectConfiguration configuration = new OpenIdConnectConfiguration();
-            TestUtilities.CallAllPublicInstanceAndStaticPropertyGets(configuration, "OpenIdConnectMetadata_GetSets");
+            Type type = typeof(OpenIdConnectConfiguration);
+            PropertyInfo[] properties = type.GetProperties();
+            if (properties.Length != 39)
+                Assert.True(false, "Number of properties has changed from 39 to: " + properties.Length + ", adjust tests");
 
-            List<string> methods = new List<string> { "AuthorizationEndpoint", "EndSessionEndpoint", "Issuer", "JwksUri", "TokenEndpoint", "UserInfoEndpoint" };
-            List<string> errors = new List<string>();
-            foreach(string method in methods)
-            {
-                TestUtilities.GetSet(configuration, method, null, new object[] { Guid.NewGuid().ToString(), null, Guid.NewGuid().ToString() }, errors);
-            }
+            TestUtilities.CallAllPublicInstanceAndStaticPropertyGets(configuration, "OpenIdConnectConfiguration_GetSets");
+
+            GetSetContext context =
+                new GetSetContext
+                {
+                    PropertyNamesAndSetGetValue = new List<KeyValuePair<string, List<object>>>
```

Comment:
This is consistent with the way we declare that everywhere else:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/KDev/test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs#L55

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/KDev/test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectMessageTests.cs#L115

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/KDev/test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectProtocolValidationContextTests.cs#L41